platforms/android/tangram/src/main/cpp/jniExports.cpp

```diff
@@ -515,19 +512,12 @@ extern "C" {
             jniEnv->DeleteLocalRef(value);
         }
 
-        if (sceneUpdates.empty()) { return; }
+        // Already checked on the java side
+        if (sceneUpdates.empty()) { return -1; }
```

Comment:
It looks like we already handle this in the java side (also from your comment), so probably no use to have this check here.